browser/unstoppable_domains/test/BUILD.gn

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/build/config.gni")
+import("//brave/components/unstoppable_domains/buildflags/buildflags.gni")
+import("//testing/test.gni")
+
+source_set("browser_tests") {
+  testonly = true
+  defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
+  if (unstoppable_domains_enabled) {
```

Comment:
I think there is no need to check it. The whole build.gn shouldn't be used if the flag is off

Reply:
Fixed in https://github.com/brave/brave-core/pull/7702/commits/62489746e5f89543159209c4c35f0f12ba111a21